src/main/kotlin/org/kamiblue/client/module/modules/client/CommandConfig.kt

```diff
@@ -15,16 +19,25 @@ internal object CommandConfig : Module(
     showOnArray = false,
     alwaysEnabled = true
 ) {
-    val prefix = setting("Prefix", ";", { false })
-    val toggleMessages = setting("Toggle Messages", false)
+    var prefix by setting("Prefix", ";", { false })
+    val toggleMessages by setting("Toggle Messages", false)
```

Comment:
Should this setting not be private?

Reply:
`;toggle` command needs it to not send duplicate message